hack/test-integration.sh

```diff
@@ -8,4 +8,4 @@ if [[ -z "${OPENSHIFT_SKIP_BUILD-}" ]]; then
 fi
 os::util::environment::update_path_var
 
-COVERAGE_SPEC=" " DETECT_RACES=false TMPDIR="${BASETMPDIR}" TIMEOUT=45m GOTESTFLAGS="-sub.timeout=120s" "${OS_ROOT}/hack/test-go.sh" "test/integration/runner"
+COVERAGE_SPEC=" " DETECT_RACES=false TMPDIR="${BASETMPDIR}" TIMEOUT=45m GOTESTBINARY_FLAGS="-sub.timeout=120s -sub.run=${1:-}" "${OS_ROOT}/hack/test-go.sh" "test/integration/runner"
```

Comment:
Pass the flag to test-go, not to the sub run.

Reply:
+1, `${1:+-run TestIntegration/}${1:-}`